VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -555,6 +555,20 @@ describe('ParticipantsPanelComponent', () => {
         component.toggleMuteParticipant(pat);
         expect(videocallService.muteParticipant).toHaveBeenCalledWith(pat.pexipId, false, component.conferenceId, pat.id);
     });
+    describe('handleParticipantMediaStatusChange', () => {
+        it('should call updateParticipant for a linked participant witha hearing role interpreter', () => {
+            // Arrange
+            const mediaStatus = new ParticipantMediaStatus(true, false);
+            const interpreter = component.participants.filter(x => x instanceof LinkedParticipantPanelModel)[0];
+            const message = new ParticipantMediaStatusMessage(conferenceId, interpreter.id, mediaStatus);
```

Comment:
interpreter.id will not work as we need to provide one of the participants ids not the id of the LinkedParticipantPanelModel

Reply:
removed